contentcuration/contentcuration/settings.py

```diff
@@ -15,17 +15,18 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '_s0k@&o%m6bzg7s(0p(w6z5xbo%vy%mj+xx(w3mhs=f0ve0+h2'
+SECRET_KEY = '_s0k@&o%m6bzg7s(0p(w6z5xbo%vy%mj+xx(w3mhs=f0ve0+h2' # TODO(aron): generate secret key, secretly!
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]  # In production, we serve through a file socket, so this is OK.
```

Comment:
Right now nginx connects through a unix socket, through this line:
```
  server unix:/var/www/{{ content_curation_prefix }}/gunicorn.sock fail_timeout=0;
```

So in my mind `allowed_hosts` shouldn't matter much, since the only way to connect to the app server is through accessing the filesystem. Either way we can change this once we have an actual domain to put here.